TestAssets/TestProjects/TestAppWithRuntimeOptions/project.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Yes we have mixed project.jsons with runtimeOptions but also targetting other concerns. 

For the purpose of testing runtimeOptions only, I think it's appropriate to have a targetted test asset.